xml/article_sap_trento.xml

```diff
@@ -0,0 +1,1046 @@
+                    </imageobject>
+                </mediaobject>
+            </figure>
+            <para>
+                The page provides the following information:
+            </para>
+            <itemizedlist>
+                <listitem>
+                    <para>
+                        The SID, a description, assigned databases (RDBMS), tenant, DB IP address, and optional tags that you can manually set.
+                        Clicking the arrow icon left to the SID name lists instances that are part of the &sap; systems, including database instances.
+                        The view provides the following information: the SID, features (processes), instance number,
+                        the system replication role and the status, the cluster name (when applicable), and the corresponding host name.
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>
+                        Filters the list by SID and tags.
+                    </para>
+                </listitem>
+            </itemizedlist>
+
+            <para>
+                Clicking on a <guimenu>SID</guimenu> entry, opens the <guimenu>SAP Systems</guimenu>
+                details view that displays the following:
+            </para>
+            <itemizedlist>
+                <title>&sap; System details</title>
+                <listitem>
+                    <para>the <guimenu>Name</guimenu> and <guimenu>Type</guimenu> of this &sap; System.</para>
+                </listitem>
+                <listitem>
+                    <para>
+                        <guimenu>Layout</guimenu>:
+                        lists the host name, the instance, the available features, its HTTP port, its
+                        start priority, and the status.
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>
+                        <guimenu>Hosts</guimenu>:
+                        lists the host name, the IP address, the cloud provider, the cluster name, the &sap;
+                        System where it belongs, and the &t.agent; version.
+                        When you click the host name, it shows you the details described in <xref
+                            linkend="lst-trento-web-hosts-details"/>.
+                    </para>
+                </listitem>
+            </itemizedlist>
+        </section>
+
+        <section>
+            <title>Showing &hana; databases</title>
+            <para>
+                To get an overview of detected &hana; databases, click the <guimenu>HANA databases</guimenu>
+                entry in the left sidebar.
+            </para>
+            <figure>
+                <title>HANA databases</title>
+                <mediaobject>
+                    <imageobject>
+                        <imagedata width="80%" fileref="trento-web-sapsystems.png"/>
```

Comment:
Wrong picture. It belongs to the SAP System detail view. Please include the right screenshot.

Reply:
Fixed now.